entries/a/artists.spotify.com.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Spotify for Artists": {
+    "domain": "spotify.com",
+    "url": "artists.spotify.com",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://artists.spotify.com/en/help/article/2-step-verification",
+    "categories": [
+      "entertainment"
```

Comment:
This might also fit into the `creativity` category?

Reply:
Or only the creativity category. Not sure if Artists find their Spotify dashboard to be entertainment 😁